components/org.wso2.carbon.identity.oauth2.grant.rest.endpoint/src/gen/java/org/wso2/carbon/identity/oauth2/grant/rest/endpoint/AuthStepsApi.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

Comment:
We shouldn't change copyright year of existing files. If it is a frequently changing file, we can use the year range.
Check other places as well

Reply:
@ThaminduDilshan this is within the `gen` and it was auto generated source code, as per the API spec. Hence it will take the current year. [1]

[1] - https://github.com/madurangasiriwardena/openapi-generator-cxf-wso2